packages/react-core/package.json

```diff
@@ -7,7 +7,7 @@
   "exports": {
     ".": {
       "import": "./dist/esm/index.js",
-      "require": "./dist/index.js"
+      "require": "./dist/index.js",
```

Comment:
Curious why the conflict could happen here? But it looks good to me

Reply:
Hm not sure, the conflict was between 

main
```
"import": "./dist/esm/index.mjs",
      "require": "./dist/index.js",
      "types": "./dist/types/index.d.ts"
```

studio/main
```
"import": "./dist/esm/index.js",
      "require": "./dist/index.js"
```

I tried fixing in the github merge tool but missed the comma

Reply:
Weird. That looks like an addition to me. 🤣